cutter/cutter/main.py

```diff
@@ -617,7 +618,7 @@ def main(
 			conn = dbmanager.get_conn()
 		except Exception:
 			delay = common.jitter(10)
-			logging.info('Cannot connect to database. Retrying in {:.0f} s'.format(delay))
+			logging.warning('Cannot connect to database. Retrying in {:.0f} s'.format(delay), exc_info=True)
```

Comment:
Should the exc_info=True be in other places where we might expect DB connection errors?

Reply:
I think I just missed this one. It was intended to have an exc_info=True all along, I just forgot.